list/PageableList.js

```diff
@@ -461,7 +450,7 @@ define([
 			var renderer = this._getFirstVisibleRenderer();
 			var nextRenderer = renderer.nextElementSibling;
 			if (this.navigatedDescendant) {
-				if (renderer && this._previousPageLoader && this._previousPageLoader.loading) {
+				if (renderer && this._busy) {
```

Comment:
I don't really understand the logic here (either before or after your change) but is there a reason to change the logic to say `this._busy` rather than `this._previousPageLoader.loading`?